lib/credit_card_widget.dart

```diff
@@ -280,13 +280,16 @@ class _CreditCardWidgetState extends State<CreditCardWidget>
         crossAxisAlignment: CrossAxisAlignment.start,
         children: <Widget>[
           if (widget.bankName != null && widget.bankName!.isNotEmpty)
```

Comment:
Suggestion : We can make extension for checking null and not empty at same time . Like this,

extension NullableStringExtension on String? {

bool get isNullOrEmpty => this == null || (this?.isEmpty ?? false);

bool get isNotNullAndNotEmpty => this != null && (this?.isNotEmpty ?? false);
}